README.md

```diff
@@ -2,11 +2,11 @@
 
 [Exchange Union](https://www.exchangeunion.com/) (XU) is a decentralized exchange layer built on the [Lightning](http://lightning.network/) and [Raiden](https://raiden.network/) networks to enable trustless and instant cryptocurrency swaps and order fulfillment between exchanges.
 
-This repo contains the early stages of an XU node implementation which encompasses the following components:
+This repo contains the early stages of the node implementation 'xud' (Exchange Union Daemon) which encompasses the following components:
 
 * Integration with [lnd](https://github.com/lightningnetwork/lnd) and [raiden](https://github.com/raiden-network/raiden) nodes.
 * Orderbook data stored in a local mysql/mariadb database.
-* Peer-to-peer networking with other XU nodes via TCP.
+* Peer-to-peer networking with other xuds via TCP.
```

Comment:
I think "XU nodes" sounds better in this context even with the name change. As an analogy, in the bitcoin world we would say "networking with other bitcoin nodes" and not "networking with other bitcoinds."